xla/tests/build_defs.bzl

```diff
@@ -111,7 +111,7 @@ def xla_test(
         if backend == "cpu":
             backend_deps = ["//xla/service:cpu_plugin"]
             backend_deps += ["//xla/tests:test_macros_cpu"]  # buildifier: disable=list-append
-        elif backend == "gpu":
+        elif backend == "gpu" or backend == "gpu_v100":
```

Comment:
It looks to me that this belongs into a separate PR (I think I have even already seen it floating around)